modules/test/playwright/tests/frontend-data-set-views-web/visualizationModes.spec.ts

```diff
@@ -563,123 +602,83 @@ fragmentTest.describe('Visualization Modes in the fragment', () => {
-	const SORTABLE_COLUMN_INDEX = 5;
-
-	await test.step('Navigate to table visualization mode page', async () => {
-		await visualizationModesPage.goto({
-			dataSetLabel,
-			viewLabel,
-		});
-
-		await visualizationModesPage.selectTab('Table');
-
-		await expect(
-			visualizationModesPage.page.getByPlaceholder('Search')
-		).toBeVisible();
-	});
-
-	await test.step('Add fields', async () => {
-		await visualizationModesPage.openAddFieldsModal();
-
-		await visualizationModesPage.addRootField(SAMPLE_SCALAR_FIELD);
-		await visualizationModesPage.addRootField(SAMPLE_OBJECT_FIELD);
-		await visualizationModesPage.addChildField(
-			[SAMPLE_OBJECT_FIELD],
-			SAMPLE_OBJECT_CHILD_FIELD
-		);
 
-		await visualizationModesPage.saveAddFieldsModal();
-	});
-
-	await test.step('Check if field defaults are correct', async () => {
-		await expect(
-			visualizationModesPage
-				.getRowByText(SAMPLE_SCALAR_FIELD)
-				.locator('td')
-				.nth(SORTABLE_COLUMN_INDEX)
-		).toHaveText('true');
-
-		await expect(
-			visualizationModesPage
-				.getRowByText(`${SAMPLE_OBJECT_FIELD}.*`)
-				.locator('td')
-				.nth(SORTABLE_COLUMN_INDEX)
-		).toHaveText('false');
-
-		await expect(
-			visualizationModesPage
-				.getRowByText(
-					`${SAMPLE_OBJECT_FIELD}.${SAMPLE_OBJECT_CHILD_FIELD}`
-				)
-				.locator('td')
-				.nth(SORTABLE_COLUMN_INDEX)
-		).toHaveText('true');
-	});
-
-	await test.step('Edit a field', async () => {
-		await visualizationModesPage
-			.getRowByText(SAMPLE_SCALAR_FIELD)
-			.locator('.actions-cell button')
-			.click();
-
-		const editButton = visualizationModesPage.page.getByRole('menuitem', {
-			name: 'Edit',
-		});
-
-		await expect(editButton).toBeInViewport();
-
-		await editButton.click();
-
-		const sortableInput =
-			visualizationModesPage.page.getByLabel('Sortable');
-
-		await expect(sortableInput).toBeInViewport();
-		await expect(sortableInput).toBeEnabled();
-		await expect(sortableInput).toBeChecked();
-
-		await sortableInput.click();
-
-		await expect(sortableInput).not.toBeChecked();
-
-		await visualizationModesPage.saveAddFieldsModal();
-
-		await expect(
-			visualizationModesPage
-				.getRowByText(SAMPLE_SCALAR_FIELD)
-				.locator('td')
-				.nth(SORTABLE_COLUMN_INDEX)
-		).toHaveText('false');
-	});
+	fragmentTest(
+		'Show mapped scalar array field in the fragment @LPD-11769',
+		async ({
+			apiHelpers,
+			dataSetManagerApiHelpers,
+			fdsFragmentPage,
+			page,
+			site,
+		}) => {
+			const SAMPLE_SCALAR_ARRAY_FIELD = 'keywords';
+			const SAMPLE_SCALAR_ARRAY_CONTENT = ['one', 'two', 'three'];
 
-	await test.step('Check if object field has disabled sortable option', async () => {
-		await visualizationModesPage
-			.getRowByText(`${SAMPLE_OBJECT_FIELD}.*`)
-			.locator('.actions-cell button')
-			.click();
+			await fragmentTest.step('Create table fields', async () => {
+				await dataSetManagerApiHelpers.createDataSetViewFields({
+					extraBodyParams: {
+						keywords: SAMPLE_SCALAR_ARRAY_CONTENT,
```

Comment:
you create the DSM field that allows to test this and populate it with the data you want to show in the fragment, all at once! Hard to be more efficient!